debugging/coreclr/netCoreProjectProvider.ts

```diff
@@ -2,13 +2,12 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
-import * as path from 'path';
 import { DotNetClient } from "./dotNetClient";
 import { FileSystemProvider } from "./fsProvider";
 import { TempFileProvider } from './tempFileProvider';
 
 const getTargetPathProjectFileContent =
-`<Project>
+    `<Project>
```

Comment:
Why this change?  Personally I prefer them all to be left-indented if any part of the multi-line string is.

Reply:
My only change was to remove the `path` import above. This must have been auto-formatted

Reply:
Yeah this is getting changed on-save. I like it better left-indented, too, but don't feel like fighting the auto format